src/main/java/core/basesyntax/Default.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Default implements Figure {
+    private String color;
+
+    public Default(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
+
+    @Override
+    public double getProperty() {
+        return 0;
+    }
+
+    @Override
+    public void draw() {
+
+    }
```

Comment:
The 'draw' method is empty. If this class is not supposed to implement drawing, consider throwing an UnsupportedOperationException or providing a meaningful implementation.